internal/lm/resource.go

```diff
@@ -302,3 +303,15 @@ func getArchFamily(computeMajor, computeMinor int) string {
 	}
 	return "undefined"
 }
+
+func sanitise(input string) string {
+	var sanitised string
+	sanitised = strings.ReplaceAll(input, "(", "")
+	sanitised = strings.ReplaceAll(sanitised, ")", "")
+
+	// remove redundant blank spaces
+	sanitised = strings.Join(strings.Fields(strings.TrimSpace(sanitised)), " ")
+
+	sanitised = strings.ReplaceAll(sanitised, " ", "-")
```

Comment:
Looking at `strings.Fields` https://pkg.go.dev/strings#Fields I think that the `strings.TrimSpace` is redundant. We could simplify the following to:
```suggestion
	sanitised = strings.Join(strings.Fields(sanitised), "-")
```

Reply:
Done, thanks